ts/@live-compositor/core/src/compositor.ts

```diff
@@ -14,6 +14,7 @@ export async function createLiveCompositor(manager: CompositorManager): Promise<
 export class LiveCompositor {
   private manager: CompositorManager;
   private api: ApiClient;
+  private store: ContextStore = new ContextStore();
```

Comment:
[NIT]
Personal opinion here, but `store` as field name sounds weird. It doesn't indicate what is "stored" there. I would use `context`.

Reply:
this is store, context it's what returned from the hooks

Reply:
What do you mean by `store`? Every data structure can be referred to as a `store`. Every struct, map, list, etc. "stores" sth. My point is that this name doesn't indicate what is stored there. 

When you see:
```
compositor.store
```
It's impossible to tell what is in `store` without checking this `LiveCompositor` class definition/field type.

Reply:
- this internal implementation, this is not user facing
- store is well established name in react and probably even in general in web ecosystem